README.md

```diff
@@ -1,33 +1,36 @@
 <img src="External/ReadMeImages/MRTK_Logo_Rev.png">
 
-# What is MixedRealityToolkit-vNext
-The V1 Mixed Reality Toolkit is a collection of scripts and components intended to accelerate development of applications targeting Microsoft HoloLens and Windows Mixed Reality headsets.
+# What is MixedRealityToolkit
 
-### For details on the different branches used in the Mixed Reality Toolkit repositories, check this [Branch Guide here](https://github.com/Microsoft/MixedRealityToolkit-Unity/wiki/Branch-Guide).
+MRTK is a Microsoft Driven opensource project. 
 
-This new version of the MixedRealityToolkit aims to further extend the capabilities of the toolkit and also introduce new features, including the capability to support more VR/AR/XR platforms beyond Microsoft's own Mixed Reality setup.
+MRTK-Unity provides a set of foundational components and features to accelerate MR app development in Unity. Latest Release of MRTK (V2) supports HoloLens/HoloLens 2, WMR, and OpenVR platform. Start your project with MRTK, and 
 
-The vNext branch is taking all the best lessons learned from the original Mixed Reality Toolkit and refactoring / restructuring it to both:
+# What's MRTK-Unity good for 
+ 
+* Provide basic features as an easy to use SDK to reduce the barrier-to-entry to get started.
 
-* Support a wider audience, allowing solutions to be built that will run on multiple VR / AR / XR platforms such as Mixed Reality,  Steam/Open VR and OpenXR (initially)
+* Enable rapid prototyping by providing the basic building blocks for MR app development. 
 
-* Provide an easier to use SDK, to enable rapid prototyping and ease adoption for new users (or users of previous frameworks)
+* Showcase best practices in MR with UI controls and interactions that matches the WMR and HoloLens Shell. 
```

Comment:
Nit: General question about the usage of three letter acronyms vs fully expanding the first reference (i.e. Windows Mixed Reality (WMR)) - if we think that contributors are expected to already know about all of these acronyms that's fine, I just find that it's easier to expand things out in the general README and then use those abbreviations deeper in the docs)